lib/neurx/activation_functions/activators.ex

```diff
@@ -28,14 +28,19 @@ defmodule Neurx.Activators do
   Sigmoid function. See more at: https://en.wikipedia.org/wiki/Sigmoid_function
   """
   def sigmoid(input) do
-    1 / (1 + :math.exp(-input))
+    if -input > 705 do
+      1 / (1 + :math.exp(705))
```

Comment:
705 seems like a rather arbitrary number??

Reply:
exp(710) throws an error as it is larger than the max float size of 1.7976931348623157e+308.

705 was arbitrarily chosen as the softmax for this function.

Reply:
alrighty, could we include a comment to describe why this number was chosen?